indexBasecodeANSWER1c.html

```diff
@@ -0,0 +1,157 @@
+                              <p>Credit card type
+                              <input type=Radio name="CC" value="Visa" checked>Visa
+                              <input type=Radio name="CC" value="Amex">Amex
+                              <input type=Radio name="CC" value="Mastercard">Mastercard
+                              </p>
+                              <p>Credit card number
+                              <input type=Password id="cardnumber" >
+                              </p>
+                              <p><nobr><input type='submit' value="Buy">
+                              <input type='reset' value="Clear"></nobr></p>
+                        </form></p>
+                  </div>  <!-- end col1content -->
+            </div>  <!-- end col1 -->
+            <div id="col2">
+                  <div id="col2content">
+                  <h3 id='intro'>Introduction </h3>
+                  <p id='blurb'>
+                  Do you want to be amongst the first space flight passengers
+                  to Mars? Join our flight service (as endorsed by Patrick
+                  Mooreish) and see the Red Planet for yourselves !!
+                  All you have to do is fill in the details (Name, Address,
+                  Phone, Email and Credit Card details) and we will rush you
+                  the tickets. This flight is SO NOT likely to happen !!!!!! (for a while)</p>
+                  </div>  <!-- end col2content -->
+            </div>  <!-- end col2 -->
+            <div id="footer">
+            <p><b>&copy; Bear Computer Services Ltd. | Original style design by BlueSky Web Studios</b></p>
+            </div>  <!-- end footer -->
+      </div>  <!-- end wrapper -->
+</body>
+<script>
+		 function Init()
+		 {
+		  	var ins = document.getElementsByTagName("input");
+		    var imgs = document.getElementsByTagName("img");
+		 
+	 		if (window.addEventListener)
+	 		{
+	 			for(i=0 ; i<ins.length ; i++){
+						ins[i].addEventListener("focus",doHighlights,false);
+	 					ins[i].addEventListener("blur",doHighlights,false);
+				}
+				
+	 			document.getElementById("blurb").addEventListener("mouseover",doBackground,false);
+	 			document.getElementById("blurb").addEventListener("mouseout",doBackground,false);
+				
+				for(i=0 ; i<imgs.length ; i++){
+	 					imgs[i].addEventListener("mouseover",doText,false);
+	 					imgs[i].addEventListener("mouseout",doText,false);
+				}
+	 		}
+	 		else if (window.attachEvent)
+	 		{
+	 			for(i=0 ; i<ins.length ; i++){
+						ins[i].attachEvent("focus",doHighlights);
+	 					ins[i].attachEvent("blur",doHighlights);
+				}
+	 			document.getElementById("blurb").attachEvent("onmouseover",doBackground);
+	 			document.getElementById("blurb").attachEvent("onmouseout",doBackground);
+				for(i=0 ; i<imgs.length ; i++){
+	 					imgs[i].attachEvent("mouseover",doText);
+	 					imgs[i].attachEvent("mouseout",doText);
+				}
+	 		}
+	 		else
+	 		{
+ 			 	alert("I have nothing");

```

Comment:
😄 Seems we are alerting for `nothing` here. The use case will be for when pressing the `buy` button when we have nothing and this will be appropriate on the validation code for the buy form I guess.